includes/class-wc-google-gtag-js.php

```diff
@@ -201,26 +260,26 @@ function gtag(){dataLayer.push(arguments);}
 	 * @return string
 	 */
 	protected function add_transaction_enhanced( $order ) {
-		// Order items
-		$items = "[";
-		if ( $order->get_items() ) {
-			foreach ( $order->get_items() as $item ) {
-				$items .= self::add_item( $order, $item );
+		$event_items = array();
+		$order_items = $order->get_items();
+		if ( ! empty( $order_items ) ) {
+			foreach ( $order_items as $item ) {
+				$event_items += self::add_item( $order, $item );
```

Comment:
I don't think this is intended here. The function `add_item` returns an array (would be nice to mention that in the docblock). So when we combine two arrays with the `+` operator we essentially do a union of the two arrays. You compensate for that later by wrapping `event_items` in another array. Which will match up as long as you are tracking only one order item, but as soon as you add multiple order items that's going to fail.
```suggestion
				$event_items[] = self::add_item( $order, $item );
```

I'd suggest to try running this code to see how that differs:

```php
$one = [ 'id' => 1, 'label' => 'One' ];
$two = [ 'id' => 2, 'label' => 'Two' ];

$numbers =  [];
$numbers += $one;
$numbers += $two;

var_dump( array( $numbers ) );

$numbers   = [];
$numbers[] = $one;
$numbers[] = $two;

var_dump( $numbers );
```